File: Dddml.Wms.Services/Domain/Warehouse/WarehouseUtils.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dddml.Wms.Domain.Warehouse
{
    public static class WarehouseUtils
    {

        public static string GetInTransitLocatorId(string warehouseId)
        {
            if (String.IsNullOrWhiteSpace(warehouseId)) { throw new ArgumentNullException("warehouseId"); }
            return warehouseId + "-IN_TRANSIT";
        }

        public static string GetReceivingLocatorId(string warehouseId)
        {
            if (String.IsNullOrWhiteSpace(warehouseId)) { throw new ArgumentNullException("warehouseId"); }
            return warehouseId + "-RECEIVING_AREA";
        }

    }
}

File: java/Dddml.Wms.JavaServices/src/main/java/org/dddml/wms/domain/warehouse/WarehouseUtils.java
package org.dddml.wms.domain.warehouse;

/**
 * Created by yangjiefeng on 2018/2/2.
 */
public class WarehouseUtils {

    public static String getInTransitLocatorId(String warehouseId) {
        if (warehouseId == null) {
            throw new IllegalArgumentException("Null warehouseId.");
        }
        return warehouseId + "-IN_TRANSIT";
    }

    public static String getReceivingLocatorId(String warehouseId) {
        if (warehouseId == null) {
            throw new IllegalArgumentException("Null warehouseId.");
        }
        return warehouseId + "-RECEIVING_AREA";
    }

}
